tests/test_crypto.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""Tests for `tiingo` package."""
+
+from unittest import TestCase
+
+from tiingo import TiingoClient
+
+
+# CRYPTO ENDPOINTS
```

Comment:
These test cases are a great start. Would it be possible to take a pass at writing the test in a way that we add a new fixture to this folder  `https://github.com/hydrosquall/tiingo-python/tree/master/tests/fixtures`? This way, people can see a sample of what the returned data is meant to look like. 

Feel free to let me know if you have any questions about how to do this.

Reply:
Alright, will add the fixtures ;)

Reply:
I have a problem with getting responses from `api.tiingo.com`! I think my token is rate limited, but other than that, when I make an API call to `crypt/top/` price and set a `endDate` querystring parameter, I get a 500 server error. I don't know it is raising an exception in tiingo's webserver or not.

Reply:
Thank you for the great work @n1rna! Please make sure to reach out to me get a free dev account!

You helped us catch an error that was related to older top of book data. We deprecated the "endDate" parameter for top-of-book since nobody seemed to use it and we plan to make order book tick data available shortly. This should help users reconstruct historical top-of-book using the tick data and promote transparency.

Reply:
Thanks!
So do I need to remove `endDate` parameter from the API client and also the test cases in this PR?